Drop keylime example (#3523)
kairos-io/kairos: examples/keylime/README.md

# Setting up the keylime agent in Kairos

This example how to use Keylime with Kairos in order to provide verified measurement for runtime binaries or files in a Kairos system. This, for instance involve having measurements for specific files in the persistent portion of the disk, or the configuration directories.

Most of the steps are already covered in the [Keylime documentation](https://keylime-docs.readthedocs.io/en/latest/). Here we will cover the steps that are specific to Kairos.

## Extend Kairos

> First of all we need to create a Kairos derivative with the keylime agent, in order to do this we use the [Kairos factory process](https://kairos.io/docs/reference/kairos-factory/) where we will build our own OS as a container image and we will install the kairos-agent:

Author: ```suggestion
First of all we need to create a Kairos derivative with the keylime agent, in order to do this we use the [Kairos factory process](https://kairos.io/docs/reference/kairos-factory/) where we will build our own OS derivative to bundle keylime-agent:
```

```Dockerfile

> FROM quay.io/kairos/rockylinux:9-core-amd64-generic-v3.4.1 AS base
> RUN dnf -y update && dnf -y install --setopt=install_weak_deps=False keylime && dnf clean all

Author: ```suggestion

# Build the keylime agent
FROM ubuntu:24.04 AS keylime-build
RUN apt-get update && apt-get install -y --no-install-recommends git gcc libclang-dev libssl-dev libtss2-dev libzmq3-dev pkg-config rustup make && rm -rf /var/lib/apt/lists/*
WORKDIR /keylime
RUN git clone --depth 1 --branch v0.2.7 https://github.com/keylime/rust-keylime.git /keylime
ENV DESTDIR=/keylime-output
RUN rustup default stable && make && make install

# Build the final image with the keylime agent copied over, only systemd supported for now
FROM quay.io/kairos/rockylinux:9-core-amd64-generic-v3.4.1 AS default
COPY --from=keylime-build /keylime-output/ /
```

> Most of the steps are already covered in the [Keylime documentation](https://keylime-docs.readthedocs.io/en/latest/). Here we will cover the steps that are specific to Kairos.

Reviewer: ```suggestion
This example how to use Keylime with Kairos in order to provide verified measurement for runtime binaries or files in a Kairos system. This, for instance involve having measurements for specific files in the persistent portion of the disk, or the configuration directories.

Most of the steps are already covered in the [Keylime documentation](https://keylime-docs.readthedocs.io/en/latest/). Here we will cover the steps that are specific to Kairos.

## Extend Kairos

First of all we need to create a Kairos derivative with the keylime agent, in order to do this we use the [Kairos factory process](https://kairos.io/docs/reference/kairos-factory/) where we will build our own OS as a container image and we will install the kairos-agent:
```
